code/modules/jobs/job_types/chaplain.dm

```diff
@@ -18,6 +18,11 @@
 
 	family_heirlooms = list(/obj/item/toy/windup_toolbox, /obj/item/reagent_containers/food/drinks/bottle/holywater)
 
+	mail_goodies = list(
+		/obj/item/reagent_containers/food/drinks/bottle/holywater = 30,
```

Comment:
holy water is a really underwhelming gift for chaplains; consider replacing this with unholy water or something

Reply:
you could probably also add candle packs and matchboxes as filler here

ooh, and maybe some of those flowers that are in the chapel on some maps

if you wanted to push this system a little bit, you could add a very slim chance (the same odds as a cargo tech getting a mosin) of getting a(nother) spellbook of lesser smoke or an old soulstone (the single-use kind).